frb_example/pure_dart/dart/lib/main.dart

```diff
@@ -130,6 +130,45 @@ void main(List<String> args) async {
     expect(cnt, 10);
   });
 
+  test('dart call handle_stream_1', () async {
+    final max = 4;
+    final key = 3;
+    final stream = api.handleStream1(key: key, max: max);
+    var cnt = 0;
+    await for (final value in stream) {
+      print("output from handle_stream_1's stream: $value");
+      expect(value.key, key);
+      cnt++;
+    }
+    expect(cnt, max);
+  });
+
+  test('dart call handle_stream_2', () async {
+    final max = 6;
+    final key = 1;
+    final stream = api.handleStream1(key: key, max: max);
```

Comment:
Oops maybe this is "2" not "1"?

Reply:
this was just a random number to prove the other argument is being passed and we get it back

Reply:
No, I mean you call `api.handleStream1`, but you want to call `api.handleStream2`

Reply:
No, I mean you call `api.handleStream1`, but you want to call `api.handleStream2`